File: boot/api/src/main/java/min/micro/api/user/service/UserService.java
package min.micro.api.user.service;

import min.micro.api.user.domain.UserDto;
import min.micro.api.user.domain.UserVo;

import java.util.List;

public interface UserService {

    String signup(UserVo user);
    UserDto signin(UserVo user);
    List<UserVo> findAll();
    List<UserDto>fetch(UserVo user);

}

File: react/src/users/service/user.service.js
import axios from 'axios'

const findAll=()=>{
    return axios.get("http://localhost:8080/users/findAll")
}

const signUp=(user)=>{
    
    return axios.post("http://localhost:8080/users/signup", user)
}

export default {findAll,signUp}
